src/dispatch/plugins/dispatch_slack/events.py

```diff
@@ -193,7 +193,7 @@ def handle_reaction_added_event(
 
 
 @slack_background_task
-def increment_activity(
+def increment_participant_activity(
```

Comment:
Should we examine the activities that increment [Participant.activity](https://github.com/Netflix/dispatch/blame/master/src/dispatch/participant/models.py#L48)?

Reply:
Activities in and outside the incident channel other than messages? I haven't given it a lot of thought, but if you have ideas on other activities we could programmatically track let me know.

Reply:
Document accesses (opens) or changes might be a useful signal. If you're interacting with an incident, I think you're very likely to hit the documents.

Reply:
Yeah, we could look into it, but history tells us that the drive activity api is not very reliable.